package.json

```diff
@@ -228,14 +228,14 @@
     "worker-loader": "2.0.0"
   },
   "scripts": {
-    "build": "cross-env NODE_ENV=production webpack --config webpack_config/production.js",
-    "build:staging": "cross-env NODE_ENV=staging webpack --config webpack_config/staging.js",
+    "build": "cross-env TARGET_ENV=default webpack --config webpack_config/production.js",
```

Comment:
Maybe 'production' to be more explicit? Or 'prod'? (I should have better named the schema)